tests/foreman/ui/test_syncplan.py

```diff
@@ -53,6 +53,7 @@ def validate_repo_content(repo, content_types, after_sync=True):
             ), 'Repository contains invalid number of content entities.'
 
 
+@pytest.mark.e2e
```

Comment:
I propose selecting a comprehensive end-to-end test that applies the sync plan and verifies the synchronization of the repository based on the expected timings.